packages/wdio-sync/tests/wrapCommand.test.ts

```diff
@@ -198,12 +203,12 @@ describe('wrapCommand:runCommand', () => {
      */
     describe('WDIO_TIMER', () => {
         it('WDIO_TIMER listener', () => {
-            process.emit('WDIO_TIMER', { id: 1, start: true })
-            process.emit('WDIO_TIMER', { id: 1 })
+            (process.emit as Function)('WDIO_TIMER', { id: 1, start: true });
```

Comment:
let's define a variable for `process.emit ` to not cast it so many times

Reply:
Good idea! done